cpp/ql/test/library-tests/syntax-zoo/raw_consistency.expected

```diff
@@ -117,21 +64,22 @@ backEdgeCountMismatch
 useNotDominatedByDefinition
 | VacuousDestructorCall.cpp:2:29:2:29 | Address | Operand 'Address' is not dominated by its definition in function '$@'. | VacuousDestructorCall.cpp:2:6:2:6 | void CallDestructor<int>(int, int*) | void CallDestructor<int>(int, int*) |
 | misc.c:219:47:219:48 | Address | Operand 'Address' is not dominated by its definition in function '$@'. | misc.c:219:5:219:26 | int assign_designated_init(someStruct*) | int assign_designated_init(someStruct*) |
+| ms_assume.cpp:11:30:11:33 | Address | Operand 'Address' is not dominated by its definition in function '$@'. | ms_assume.cpp:11:12:11:12 | int f(int, char*[]) | int f(int, char*[]) |
```

Comment:
This seems to be the only new result that crops up here.

Reply:
Yeah, I don't think we handle `__assume` correctly in the IR right now?

Reply:
That could be. I didn't look at this in detail. I'd assume (pardon the pun) the problem was already there but somehow hidden in all function name confusion.

Reply:
Oh wait, that should be handled somewhat okay: https://github.com/github/codeql/blob/3afb9c1b3bf2a6177e8cb3148cfc6218b6a9c298/cpp/ql/lib/semmle/code/cpp/ir/implementation/raw/internal/TranslatedExpr.qll#L3215

Reply:
> That could be. I didn't look at this in detail. I'd assume (pardon the pun) the problem was already there but somehow hidden in all function name confusion.

Yeah, I'm sure this PR can't be blamed for this. Thanks for cleaning up all those inconsistencies!